package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "top-video-player",
+  "version": "",
+  "description": "",
+  "author": "",
+  "main": "index.js",
+  "license": "",
+  "engines": {
+    "node": ">=6.13.0"
+  },
+  "scripts": {
+    "seed": "node server/seeder.js",
+    "build": "webpack --config webpack.config.js",
```

Comment:
If you have a webpack.config.js, webpack will run that config file automatically, so you don't need to specify in the script. Not a big deal. Just FYI :)

Reply:
Oh really? Sweeeeet!